src/chunk_iterator.rs

```diff
@@ -61,7 +63,8 @@ where
     T: BufRead,
 {
     #[allow(dead_code)] // Temporary while no consumer of this API
-    /// Get a new iterator ready to read `chunk_size` bytes from `reader`, representing
+    /// Get a new iterator ready to read `chunk_size` bytes from `reader`,
+    /// representing
```

Comment:
Did a tool of yours refactor/update these for you? If so, what tool? because Emacs and `cargo fmt` went "whatever" on all of these!

... Or is this one of the tricks of using a rust-fmt config file, which I don't have right now?
I may want to learn about rust-fmt config a bit more, maybe even bump MSRV to match...

Reply:
i just ran `cargo fmt` in your project.
there's a `.rustfmt` file in your project root which determines this behaviour

this is

```toml
wrap_comments = true
```

maybe your emacs plugin is using a global config instead of the local one or something?